public/visualization.component.ts

```diff
@@ -128,161 +141,97 @@ export class VisualizationComponent {
         let x = 1;
         let y = groupBy == "none" || colorBy == "none" ? 15 : 30;
 
-        if (colorBy !== "none") {
-            this.drawLegend(colorBy);
-        }
+        // Certain entries are "fake", consolidations of multiple individuals. In certain cases, we want them to be in a separate group.
+        var separateFakes = false;
+        if (sizeBy === "travelTime" || (groupBy === "travel" || groupBy === "tours")) {
+            separateFakes = true;
+            var fakeEntries = [];
+        }        
+
         for (let i in entryGroups) {
             const group = allGroups[i];
-            const entriesInGroup = (entryGroups[i] as { entries: any[], request: any }).entries;
+            let entriesInGroup = (entryGroups[i] as { entries: any[], request: any }).entries;
 
             d3.select('svg').append("text")
                 .attr("x", x)
                 .attr("y", y)
                 .text(function (d) { return group.title; });
             y += 15;
 
-            let dotGroup = this.drawDots(entriesInGroup, colorBy, sizeBy, y);
+            if (separateFakes) {
```

Comment:
It would be nice to extract this into a separate function later to make sure it's doing the right thing... but not a priority now.